CalculatorTests/CalculatorTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,320 @@
+        [TestCase("!@£$%^&*", "!@£$%^&*", "!@£$%^&*")]
+        [TestCase("", "", "")]
+        [TestCase("0", "0", "0")]
+        [TestCase("", "20", "30")]
+        [TestCase("10", "", "30")]
+        [TestCase("10", "20", "")]
+        [TestCase("0", "20", "30")]
+        [TestCase("10", "0", "30")]
+        [TestCase("10", "20", "0")]
+        [TestCase("100001", "20", "30")]
+        [TestCase("100", "101", "30")]
+        [TestCase("100", "99", "366")]
+        public void NegativeInputTests(string depositAmount, string rateOfInterest, string investmentTerm)
+        {
+            //arrange
+            IWebElement depositAmountField = driver.FindElement(By.XPath("//tr[contains( string(), 'Deposit Amount' )]//input"));
+            IWebElement rateOfInterestField = driver.FindElement(By.Id("percent"));
+            IWebElement investmentTermField = driver.FindElement(By.Id("term"));
+            IWebElement financialYear = driver.FindElement(By.Id("finYear"));
+            IWebElement financialYearButton365 = financialYear.FindElements(By.TagName("td"))[1];
+            IWebElement calculateButton = driver.FindElement(By.Id("calculateBtn"));
+
+
+            //act
+            depositAmountField.SendKeys(depositAmount);
+            rateOfInterestField.SendKeys(rateOfInterest);
+            investmentTermField.SendKeys(investmentTerm);
+            financialYearButton365.Click();
+
+            //assert
+            Assert.IsFalse(calculateButton.Enabled);
+        }
+
+        [TestCase("1", "1", "1", "10", "March", "2020", "0.00", "1.00", "11/03/2020")]
+        [TestCase("100", "10", "15", "12", "July", "2022", "0.41", "100.41", "27/07/2022")]
+        [TestCase("50000", "50", "300", "25", "August", "2021", "20,547.95", "70,547.95", "21/06/2022")]
+        [TestCase("99999", "99", "360", "30", "September", "2012", "97,642.86", "197,641.86", "25/09/2013")]
+        [TestCase("100000", "100", "365", "20", "September", "2029", "100,000.00", "200,000.00", "20/09/2030")]
+        public void PositiveInput365daysTests(string depositAmount, string rateOfInterest, string investmentTerm, string startDay, string startMonth, string startYear, string interestEarned, string income, string endDate)
+        {
+            //arrange
+            IWebElement depositAmountField = driver.FindElement(By.Id("amount"));
+            IWebElement rateOfInterestField = driver.FindElement(By.Id("percent"));
+            IWebElement investmentTermField = driver.FindElement(By.Id("term"));
+            IWebElement startDayField = driver.FindElement(By.Id("day"));
+            IWebElement startMonthField = driver.FindElement(By.Id("month"));
+            IWebElement startYearField = driver.FindElement(By.Id("year"));
+            IWebElement financialYear = driver.FindElement(By.Id("finYear"));
+            IWebElement financialYearButton365 = financialYear.FindElement(By.XPath("//tr[@id = 'finYear']/td[text() = '365 days']/*"));
```

Comment:
You may add a `finYear` parameter to the test-case and use it in XPath. So you can merge this and next test-cases.
Check `Интерполяция строк` https://metanit.com/sharp/tutorial/7.5.php

```suggestion
            IWebElement financialYearButton = financialYear.FindElement(By.XPath($"//tr[@id = 'finYear']/td[text() = '{finYear} days']/*"));
```

Reply:
Fixed